applications/FluidDynamicsApplication/python_scripts/adjoint_monolithic_solver.py

```diff
@@ -40,44 +102,44 @@ def GetDefaultParameters(cls):
             "volume_model_part_name" : "volume_model_part",
             "skin_parts"  : [""],
             "no_skin_parts"  : [""],
-            "dynamic_tau" : 0.0,
-            "oss_switch"  : 0,
             "echo_level"  : 0,
             "time_stepping"               : {
                 "automatic_time_step" : false,
                 "time_step"           : -0.1
             },
             "consider_periodic_conditions": false,
-            "assign_neighbour_elements_to_conditions": true
+            "assign_neighbour_elements_to_conditions": true,
+            "formulation": {
+                "element_type": "vms"
+            }
         }""")
 
-        default_settings.AddMissingParameters(super(AdjointVMSMonolithicSolver, cls).GetDefaultParameters())
+        default_settings.AddMissingParameters(super(AdjointMonolithicSolver, cls).GetDefaultParameters())
```

Comment:
```suggestion
        default_settings.AddMissingParameters(super().GetDefaultParameters())
```
Not required as we no longer support Python2.